definitions/push-images/push-images-pipeline.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: tekton.dev/v1beta1
```

Comment:
This bundle shouldn't contain a Pipeline. This should be only a task to push a given image to a given registry. The Pipeline invoking this task will be the Release Pipeline for M5, which will contain multiple tasks (with this one being one of them).

Reply:
deleted it.